Framework/MASPreferencesWindowController.m

```diff
@@ -39,10 +40,7 @@ - (id)initWithViewControllers:(NSArray *)viewControllers title:(NSString *)title
     NSString *nibPath = [[NSBundle bundleForClass:MASPreferencesWindowController.class] pathForResource:@"MASPreferencesWindow" ofType:@"nib"];
     if ((self = [super initWithWindowNibPath:nibPath owner:self]))
     {
-		_viewControllers = [NSMutableArray arrayWithArray: viewControllers];
-#if !__has_feature(objc_arc)
-        _viewControllers = [_viewControllers retain];
-#endif
+		_viewControllers = RETAINOBJ([NSMutableArray arrayWithArray: viewControllers]);
```

Comment:
> does TextMate still need MRC?

It does not and have not done for ages 😄  so I would definitely not mind dropping the MRC code.